.gitignore

```diff
@@ -0,0 +1 @@
+.vscode
```

Comment:
this isnt a very helpful .gitignore.  Could you use the relevant template from https://github.com/github/gitignore

Reply:
This project doesn't really need a gitignore file now but creating a file can make it ease for future if ignores are required.